add docs for registering spawn apps (#411)
dustproject/dust: docs/pages/apps/registration.md

  }
}
```

## Register a spawn app

> Spawn apps are displayed in a list on the spawn screen and are intended to implement spawning functionality for custom spawn tiles.

Author: ```suggestion
Spawn apps are displayed on the spawn screen and should implement spawning functionality for custom spawn tiles.
```
